### PR TITLE
READY FOR REVIEW - Remove support for API v1

### PR DESCRIPTION
**Pull request by posita**
Removes support for accessing v1 endpoints.

### CODE REVIEW

File: dropbox/dropbox.py [hunk 1]
```diff
@@ -30,16 +31,26 @@
...
     RateLimitError,
 )
-from .session import (
-    API_CONTENT_HOST,
-    API_HOST,
-    API_NOTIFICATION_HOST,
-    HOST_API,
-    HOST_CONTENT,
-    HOST_NOTIFY,
-    pinned_session,
```
Comment (mekme) [inline dropbox/dropbox.py]: [tmp_15204-syncid_uuid-1623512896.txt](https://github.com/dropbox/dropbox-sdk-python/files/819917/tmp_15204-syncid_uuid-1623512896.txt)

Comment (posita) [inline dropbox/dropbox.py]: ??? 😕

**Comment by posita** [comment]
Rebased to `master`. Updated `stone` subrepo. @braincore, feel free to suggest an alternate reviewer if you don't have time.

**Comment by Christina13Garza** [comment]
I don't know how to do any of this stuff... doesn't hurt to try tho

**Comment by posita** [comment]
[Ping!](https://youtu.be/wshyX6Hw52I) :wink:

**Pull request merged**